tower-grpc-examples/src/helloworld/server.rs

```diff
@@ -55,7 +54,7 @@ pub fn main() {
                 return Err(e);
             }
 
-            let serve = h2.serve(sock);
+            let serve = server.serve_with(sock, http.clone());
```

Comment:
Does tower-hyper have a way to use `hyper::Server` here, so we can remove the bind and incoming boilerplate?

Reply:
I don't think so as there were some compat issues around makeservices and service and hypers service stuff. Though I didnt dive too deep into this.

Reply:
We could probably just back this specific functionality into tower-hyper though?

Reply:
Yea thats what I meant, the `tower_hyper::Server` type.

Reply:
Yup it follows tower_h2 in this sense.